test/models/location_test.rb

```diff
@@ -9,6 +9,18 @@ def setup
     assert @location.valid?
   end
   
+  test "rejects blank name" do
+    @location.name = ''
+    refute       @location.valid?
```

Comment:
I would recommend adding a message to help with debugging: e.g.
```ruby
    refute       @location.valid?, 'a blank name should not be valid'
```

Reply:
cool, thanks.  Showing my lack of minitest experience :)  do you use minitest or rspec at Shippit?

Reply:
We use RSpec at Shippit.

Reply:
👍